frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -216,6 +216,7 @@ class BundleDetail extends React.Component<
 
         return (
             <ConfigurationPanel
+                ref={(node) => this.scrollToNewlyOpenedDetail(node)}

```

Comment:
Instead of calling the `scrollToNewlyOpenedDetail` function directly in the `ref` function, it might be clearer to instead just store the ref and call `scrollToNewlyOpenedDetail` in `componentDidMount` (so it's clear that it's only called when the bundle detail is opened).

Reply:
it did not work if you just store the ref and call `scrollToNewlyOpenedDetail` in `componentDidMount`. It cannot be ensured that the ref is created before `componentDidMount`. I have already tried but failed.

Reply:
Ok, did you try creating the ref in the `constructor()` perhaps, instead?

If that doesn't work, please just add a comment here explaining that the ref is created only once, and that this is the only way to properly create the ref before `componentDidMount()`.

Reply:
Yes, it did not work. 
I have updated the comment.